### PR TITLE
feat: adding logo of Html of 16 | 24 | 32 | 48 | 64 | 96 | 128 | 512 square pixels

### PR DESCRIPTION
**Pull request by Honeshwar**
What i do:
1. I add all sizes logo in their corresponding folders,
2. and I change data.js and readme.md file .

Thank you 😃

### CODE REVIEW

File: data.js [hunk 1]
```diff
@@ -304,10 +304,10 @@ const dataset = [
...
   },
   {
-    title: "Html",
-    slug: "html",
-    addedBy: "Honeshwar-Thakur",
-    cardImagePath: "./RandomSize/html.png",
+    title: "MDN",
+    slug: "mdn",
+    addedBy: "dipayansarkar47",
+    cardImagePath: "./RandomSize/mdn.png",
```
Comment (Shariar-Hasan) [inline data.js:310]: resolve this issue,
your changes, overriding others commits

Comment (Honeshwar) [reply data.js:310]: okay.

Comment (Honeshwar) [reply data.js:310]: Check now, github commit it as Merge branch 'main' into developer.

**Review by Shariar-Hasan — APPROVED** [review]
all ok

**Pull request merged**